docs/advanced_guide/cloud/notebooks/text_generation/text_generation.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "# Task: Text Generation\n",
+        "\n",
+        "In this Notebook we run Archai's [Text Generation](https://github.com/microsoft/archai/tree/main/tasks/text_generation) task on Azure Machine Learning.\n",
+        "\n",
+        "Our goal is to show how to create and run jobs for each step of the task without spending lots of computing resources. Therefore, our goal is not to train a good model -- for this purpose please refer to the original task.\n",
+        "\n",
+        "## Prerequisites\n",
+        "\n",
+        "- Python 3.7 or later\n",
+        "- An Azure subscription\n",
+        "- An Azure Resource Group\n",
+        "- An Azure Machine Learning [Workspace](https://learn.microsoft.com/en-us/azure/machine-learning/quickstart-create-resources#create-the-workspace)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Requirements"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "%pip install azure-ai-ml azure-identity \n",
+        "%pip install jinja2\n",
+        "%pip install archai"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "from pathlib import Path\n",
+        "\n",
+        "from IPython.display import display, Image\n",
+        "from IPython.core.display import HTML\n",
+        "\n",
+        "from azure.ai.ml import load_job\n",
+        "\n",
+        "import archai.common.azureml_helper as aml_helper\n",
+        "import archai.common.notebook_helper as nb_helper"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### Get a handle to the workspace\n",
+        "\n",
+        "We load the workspace from a workspace [configuration file](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment#local-and-dsvm-only-create-a-workspace-configuration-file)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "ml_client = aml_helper.get_aml_client_from_file(\"../.azureml/config.json\")"
```

Comment:
I find it safe to add this print statement after this line so user can be reminded they are about to party on the right Azure resources:

```python
print(f'Using workspace: {ml_client.workspace_name} in resource group: {ml_client.resource_group_name}')
```

Reply:
Done.